Pull-requests-at-Oppia.md

```diff
@@ -0,0 +1,64 @@
+## Table of contents
+
+* [PR lifecycle](#pr-lifecycle)
+* [Draft pull requests](#draft-pull-requests)
+* [Labeling issues and pull requests](#labeling-issues-and-pull-requests)
+* [Juggling multiple pull requests](#juggling-multiple-pull-requests)
```

Comment:
Do we need to change the id name here to match the updated title?

Reply:
Also, the heading names are not updated here?

Reply:
fixed